service/src/test/java/bio/terra/workspace/pact/WdsContractVerificationTest.java

```diff
@@ -76,7 +77,7 @@
 })
 @Tag("pact-verification")
 @Provider("workspacemanager") // should match the terra chart name
-@PactBroker()
+@PactBroker(enablePendingPacts = "true", providerTags = "main")
```

Comment:
Doing this because it seems like we should to keep WSM build from falsely failing: https://docs.pact.io/pact_broker/advanced_topics/pending_pacts

Reply:
Looks good